### PR TITLE
rename "strains" -> "research," fixes #95

### LINKED ISSUE #95: change "strains" to "research" in navigation
This might break our links. Let's check if feasible. Jonathan, can you take a look?

### PR DESCRIPTION
**Pull request by JonathanReeve**

### CODE REVIEW

File: _layouts/strains.html [hunk 1]
```diff
@@ -21,7 +21,7 @@
 <div class="cats">
     {{ content }}
 
-    {% for post in site.categories[page.title] %}
+    {% for post in site.categories['strains'] %}
```
Comment (denten) [inline _layouts/strains.html]: Hmm... Any downside to hardcoding the title like this?

**Comment by denten** [comment]
Do you think this breaks any links?

File: _layouts/strains.html (hunk 1 above)
Comment (JonathanReeve) [inline _layouts/strains.html]: That wouldn't be terrible. I'll give it a try, and add it to this PR.

**Comment by JonathanReeve** [comment]
As far as I can tell, it doesn't break any links (judging from the output of the link checker).

**Pull request merged**